src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -71,4 +72,12 @@
     }
     return Collections.emptyList();
   }
+
+  /**
+   * @return {@code [--name, key1=val1, --name, key2=val2, ...]} or {@code []} if
+   *      keyValues=empty/null.
+   */
+  public static List<String> get(String name, Map<String, String> keyValues) {
+    return Args.namedKeyValues(name, keyValues);
```

Comment:
`flagPerEntry` isn't bad; but what would you suggest for the sister method that returns the entry without the flag? `namedKeyValues` is a bit confusing I agree, but when paired with `keyValues` it because a little clearer.